### ISSUE TITLE
Search inputs are missing accessible text 

### ISSUE THREAD

**Issue by bbusenius** [uchicago-library/library_website#659]
## LIB20230113-06. Search inputs are missing accessible text (Blocker)
**Issue**: There are no labels on the Articles Plus and Website search inputs. These inputs do have aria-label="..." attributes, which do not convey the purpose of the inputs. The aria-labels on each input should be updated to provide meaningful accessible text/labeling for these inputs. 

## Related materials
[Full issue description](https://docs.google.com/document/d/15B0iu5qEQOI8BH-P48pgJul1-8A5b9d6Bi6PwoQvlTc/edit#heading=h.h09tizb0aqae)
